tests/test_isrTaskLSST.py

```diff
@@ -423,6 +454,11 @@ def test_isrFlat(self):
         # processing flats, this is a good test of the interpolation.
         isr_config.doDefect = True
 
+        # Need to make sure we are not masking the negative variance
```

Comment:
I don't think the identical comment is needed on all of these, actually.  Can probably just be on the first one.